design/0003-source-metadata.md

```diff
@@ -0,0 +1,182 @@
+- Contribution Name: Source Metadata for Traffic Metrics
+- Implementation Owner: Alex Leong
+- Start Date: 2020-04-14
+- Target Date: 
+- RFC PR: 
+- Linkerd Issue: 
+- Reviewers: 
+
+# Summary
+
+[summary]: #summary
+
+When a meshed pod sends outgoing HTTP requests the Linkerd proxy records metrics
```

Comment:
I have foolishly not looked into the implementation enough for this. So, let me get a better understanding of how this works today.

## Source

- `dst_X` label is added by the proxy
- `direction=outbound` is added by the proxy
- `X` label is added by prometheus

## Destination

- `X` label is added by prometheus
- `direction=inbound` is added by the proxy

Am I correct there?

Reply:
That's correct.